scripts/php-mise.sh

```
@@ -0,0 +1,164 @@
+    local conf="$3"
+
+    if in_beta $2; then
+        ext="$2-beta"
+    fi
+
+    (
+        set -e
+
+        {
+            if [[ ! -z ${ext_opts[$ext]+x} ]]; then
+               	pecl_bin install --configureoptions="${ext_opts[$ext]}" $ext
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)